### PR TITLE
Disable save button while submitting form in settings 

### PR DESCRIPTION
**Pull request by ijreilly**
as per title

### CODE REVIEW

**Review by thaisguigon — CHANGES_REQUESTED** [review]
Shouldn't we add this to `SettingsObjectNewFieldStep2` too?

File: packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx [hunk 1]
```diff
@@ -166,7 +169,7 @@ export const SettingsObjectFieldEdit = () => {
               <SaveAndCancelButtons
                 isSaveDisabled={!canSave}
                 onCancel={() => navigate(`/settings/objects/${objectSlug}`)}
-                onSave={handleSave}
+                onSave={formConfig.handleSubmit(handleSave)}
```
Comment (thaisguigon) [inline packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx:172]: If I'm not mistaken, `handleSubmit` passes the validated formValues to `handleSave` as a parameter, so we shouldn't need to use `formConfig.getValues()` in `handleSave`?

**Review by thaisguigon — CHANGES_REQUESTED** [review]

File: packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx (hunk 1 above)
Comment (ijreilly) [reply packages/twenty-front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx:173]: indeed, thanks!

**Review by thaisguigon — APPROVED** [review]
LGTM!

**Pull request merged**